cmd/clusterctl/api/v1alpha3/metadata_type.go

```diff
@@ -46,6 +46,16 @@ type ReleaseSeries struct {
 	Contract string `json:"contract,omitempty"`
 }
 
+func (rs ReleaseSeries) newer(release ReleaseSeries) bool {
+	if rs.Major > release.Major {
```

Comment:
Can we use the semver library for the comparison? Might be better than duplicating logic here

Reply:
sure :)